x-pack/plugins/apm/ftr_e2e/cypress/e2e/power_user/diagnostics/diagnostics.cy.ts

```diff
@@ -38,24 +35,29 @@ describe('Diagnostics', () => {
         'OK'
       );
     });
-  });
 
-  describe('when importing a file', () => {
-    beforeEach(() => {
-      cy.loginAs({ username: 'elastic', password: 'changeme' });
-      cy.visitKibana('/app/apm/diagnostics/import-export');
-      cy.get('#file-picker').selectFile(
-        './cypress/e2e/power_user/diagnostics/apm-diagnostics-8.8.0-1687436214804.json'
+    it('can display documents tab for "viewer" user', () => {
```

Comment:
What about the other tabs? What is the expected behavior when the viewer user doesn't have permission to see the Index template?

Reply:
i had written a similar comment and forget to post it 👍🏼 
@sqren We are only testing the good cases here. Should we not cover scenarios where certain roles fails "should" fail to access certain pages (even though the exception is handled)

Reply:
Good point. I'll add a scenario where the viewer user accesses one of the other tabs. In that case they won't be able to see any data. I'll consider adding a callout in the UI but I want to keep the scope of this PR down.

Reply:
Updated the e2e tests with scenarios for verifying the viewer user can navigate to the other tabs without errors. Additionally I added a message about limited privileges

<img width="1441" alt="image" src="https://github.com/elastic/kibana/assets/209966/77ecc20d-68ca-4a92-a154-abdba00c49cc">